src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/WeekNum_StronglyTypedEnum.txt

```diff
@@ -0,0 +1,173 @@
+#SETUP: disable:NumberIsFloat,StronglyTypedBuiltinEnums
+
+// Blank args
+>> WeekNum(Blank(),Blank())
```

Comment:
we should include na error example. 

WeekNum(error,valid)

WeekNum(valid,error)

Easy way to generate typed-errors is:

`If(1/0, Date(1900,11,22))`